containers/ddev-dbserver/files/etc/mysql/version-conf.d/mysql_8.cnf.txt

```diff
@@ -0,0 +1,11 @@
+[mysqld]
+
+# caching_sha2_password is the default for mysql 8.0 and 8.4, no config required
+# default-authentication-plugin=caching_sha2_password
+
+# enable mysql-native-password only in case it needs to be used by php 7.3 and below
+# This is removed in MySQL 9
+mysql-native-password=1
```

Comment:
This option works only for mysql 8.4 (should be in the docs only, not here)

And correct option for mysql 8.0:

```
[mysqld]
default_authentication_plugin=mysql_native_password
```